scala3doc/src/dotty/dokka/tasty/SyntheticSupport.scala

```diff
@@ -37,14 +37,13 @@ trait SyntheticsSupport:
 
     def getAllMembers: List[Symbol] = hackGetAllMembers(using qctx)(s)
 
+    def hasValidPos = Option.when(hackHasPosition(using qctx)(s))(s).flatMap(_.pos).fold(true)(p => p.start != p.end)
```

Comment:
Shouldn't this default to `false` if option is empty?

Reply:
Yes, I've changed it to false.